vcl/extensions.vcl

```diff
@@ -11,7 +11,13 @@
 #
 
 sub hlx_type_pipeline_before {
-    set req.http.X-Dispatch-NoCache = "true";
+    # A non-production XFN header looks like: "repo--user.hlx.page"
```

Comment:
does `XFN` stand for `x-forwarded-host` ?

Reply:
```suggestion
    # A non-production XFH header looks like: "repo--user.hlx.page"
```

It does now.